google_compute_engine/compat.py

```diff
@@ -16,28 +16,36 @@
 """A module for resolving compatibility issues between Python 2 and Python 3."""
 
 import logging
-import platform
 import subprocess
 import sys
 
-# Set distro-specific utils.
-distribution = platform.linux_distribution()
+# Get the linux distribution.
```

Comment:
Please either remove this comment or move it (and update the wording) to the section where you are importing configs specific to the Linux distribution.

Reply:
removed :hocho: